qiskit_ibm_runtime/utils/backend_converter.py

```diff
@@ -26,19 +26,16 @@
     WhileLoopOp,
 )
 from qiskit.circuit.gate import Gate
-from qiskit.circuit.library.standard_gates import (
-    get_standard_gate_name_mapping,
-    RZGate,
-    U1Gate,
-    PhaseGate,
-)
-from qiskit.circuit.delay import Delay
+from qiskit.circuit.library.standard_gates import get_standard_gate_name_mapping
```

Comment:
Where is it used in this file? I expect the linter will ask too.

Reply:
This one is used in `convert_to_target` below. This line changed because I removed the use of the gates (in `is_fractional_gate`) but not this function so I wrapped it back to a single line.